### DISCUSSION TITLE
Why do I receive occasional follow requests even though my account has never been locked?

### DISCUSSION THREAD

**Discussion by ShredderWalnut** [mastodon/mastodon#17598] [Q&A] [answered]
Hello,

My account has never been locked, but somehow two accounts sent me follow requests. 

Did this happen because those two accounts are marked as "suspicious" and therefore mandated by the server to send requests to any accounts they wish to follow? E.g. because they have followed too many accounts in total, or because they followed too many accounts during a short period? I searched, but did not find such a mechanism in the documentation. Thanks.

**Comment by ClearlyClaire** [comment, marked as answer]
This should be explained in the web interface by the following message:
> Even though your account is not locked, the \<instance\> staff thought you might want to review follow requests from these accounts manually.

This occurs when the account (or their whole server) attempting to follow you is “limited” (can still post and follow/be followed, but cannot post to public timelines and only generates notifications for people following them)

**Reply by ShredderWalnut** [reply]
Thank you for your kind help!

> explained in the web interface

I was using a third-party client, [Toot!](https://apps.apple.com/app/toot/id1229021451), to view the request, which is why I did not see the detailed message. I'll keep in mind in the future to visit the web interface for the best feature parity!
